tests/foreman/api/test_provisioning.py

```diff
@@ -276,3 +277,145 @@ def test_rhel_ipxe_provisioning(
     # assert that the host is subscribed and consumes
     # subsctiption provided by the activation key
     assert provisioning_host.subscribed, 'Host is not subscribed'
+
+
+@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
+@pytest.mark.on_premises_provisioning
+@pytest.mark.rhel_ver_match('[^6]')
+def test_rhel_pxe_provisioning_fips_enabled(
+    request,
+    module_provisioning_sat,
+    module_sca_manifest_org,
+    module_location,
+    provisioning_host,
+    pxe_loader,
+    module_provisioning_rhel_content,
+    provisioning_hostgroup,
+    module_lce_library,
+    module_default_org_view,
+):
+    """Provision a host with host param fips_enabled set to true
+
+    :id: 9e016e1d-757a-48e7-9159-131bb65dc4ef
+
+    :steps:
+        1. Configure satellite for provisioning
+        2. Provision a host with host param fips_enabled
+        3. Check that resulting host is registered to Satellite
+        4. Check host is subscribed to Satellite
+
+    :expectedresults:
+        1. Provisioning with host param fips_enabled is successful
+        2. Host installs right version of RHEL
```

Comment:
`1. Provisioning with host param fips_enabled is successful`
This `expected result` should be at number 2, if we look at **[step #2](https://github.com/SatelliteQE/robottelo/pull/12695/files#diff-65d37d80e67659592375ed9ef4e982202d3737926f41a51fef986063a4396044R303)** at line 303

Reply:
steps and expectedresults aren't supposed to be alligned